client/src/containers/EditUserProfile/styles.css

```diff
@@ -109,4 +167,18 @@
     width: 200px;
     height: 200px;
     margin: 0 auto 25px auto;
+}
+
+@media (max-width: 800px) {
+    .edit-user-profile-container .form-content-wrapper {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+    }
+    .edit-user-profile-container .left-column,
+    .edit-user-profile-container .right-column {
+        width: 75%;
+        margin: auto;
+        max-width: 400px;
```

Comment:
I am not sure, but `width: 75%;` is not necessary here. You can just leave `max-width: 400px;`